prueba/src/anagram/AnagramIdentifier.java

```diff
@@ -0,0 +1,29 @@
+package anagram;
+
+public class AnagramIdentifier {
+
+	public boolean anagramDetect(String word1, String word2) {
+
+		if (word1.length() == word2.length()) {
```

Comment:
You could try to just check if the lengths are not equals and return false, so you can skip the `else` statement at the bottom
```java
if (word1.length() != word2.length()) {
  return false;
}

int j;
int countEqualLetters = 0;
...
```